lang/LangSource/PyrLexer.cpp

```diff
@@ -1762,21 +1762,12 @@ void traverseFullDepTree2()
 		} else {
 			double elapsed;
 			buildBigMethodMatrix();
-			SymbolTable* symbolTable = gMainVMGlobals->symbolTable;
-			//post("\tNumber of Symbols %d\n", symbolTable->NumItems());
-			//post("\tByte Code Size %d\n", totalByteCodes);
-			//elapsed = TickCount() - compileStartTime;
-			//elapsed = 0;
-			elapsed = elapsedTime() - compileStartTime;
-			//post("\tcompiled %d files in %.2f seconds\n",
-			//	 gNumCompiledFiles, elapsed );
 			if(numOverwrites == 1){
 				post("\nInfo: One method is currently overwritten by an extension. To see which, execute:\nMethodOverride.printAll\n\n");
 			}
 			else if(numOverwrites > 1){
 				post("\nInfo: %i methods are currently overwritten by extensions. To see which, execute:\nMethodOverride.printAll\n\n", numOverwrites);
 			}
-			//post("compile done\n");
```

Comment:
This is a nice finishing comment. Maybe change to: `post("class library compiled.\n");` ?